contracts/scripts/deploy.sh

```diff
@@ -183,6 +181,28 @@ function deploy_simple_dex {
   eval "$__resultvar='$contract_address'"
 }
 
+function deploy_wrapped_azero {
+  local  __resultvar=$1
+
+  # --- CREATE AN INSTANCE OF THE CONTRACT
+
+  cd "$CONTRACTS_PATH"/wrapped_azero
+
+  local contract_address
+  contract_address=$(cargo contract instantiate --url "$NODE" --constructor new --suri "$AUTHORITY_SEED" --skip-confirm)
+  contract_address=$(echo "$contract_address" | grep Contract | tail -1 | cut -c 14-)
+
+  echo "wrapped Azero contract instance address: $contract_address"
+
+  # --- GRANT PRIVILEGES ON THE CONTRACT
+
+  cd "$CONTRACTS_PATH"/access_control
```

Comment:
NIT: Arguably the cleanest way of cd-ing into a directory is doing a `pushd` and trapping any error with `popd`: http://redsymbol.net/articles/bash-exit-traps/

Feel free to ignore this one :)

Reply:
oh, just changing the directory back, but it probably doesn't matter in context of circle ci, so more of a random thought :D